### ISSUE TITLE
Project Wizard <Queue --> REC>

### ISSUE THREAD

**Issue by reecepacheco** [ShelbyTV/shelby-gt-web#761]
Project REC
tl;dr - change the Queue button to REC. make it more vague and expose the actions in the stream to solve 'empty room'

WHO AND WHY - who is this being built for and why will they use it? how will this feature/product get to those that it is meant for?
Who? Shelby users. 
Why? To give users a lightweight way of interacting with Shelby that could be used to 1. save/remember the video 2. acknowledge/thank the Roller 3. Recommend others see it
further, it will help build community and solve the “empty room” feeling that Shelby currently has

WHAT AND HOW - what is this supposed to look like (design)? how will it be used by the end user (ux)?
replace Queue button with REC. icon is a simple circle like a RECORD button on a VCR plus the text "REC". REC = Record OR Recommend. it’s purposely vague
if a video is REC’d, the original Roller is notified. said video will then show those who have REC'd it (showing their avatars - see wireframe attached) aiding discovery of other users within Shelby
replace Queue navigation link with REC’d. show YOUR REC's here. maintain ability to "unREC" something 

GO / NO GO - does shelby really need this? is this going to get built? 
YES. users miss the “like” functionality. users are not Rolling. Rolling is too heavy still. 

BUILD IT
Vincent/Josh/Matyus?

LAUNCH IT - with or without fanfare?
staged, IRL UX tested, improved, deployed, included in an email update to early users, then explained in onboarding for all users eventually.

ANALYZE IT - is anyone using this? why or why not? how can it be improved? what went right, what went wrong?
will be simple to see if users engage and iterate from there.
![IMG_1076](https://f.cloud.github.com/assets/282741/23082/3db3fcfc-4a23-11e2-8d06-a64ed72c07bc.jpg)
